Sources/MarathonCore/ArgumentParser.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  ArgumentParser.swift
+//  Marathon
+//
+//  Created by pixyzehn on 2017/04/09.
+//
+//
+
+import Foundation
+
+public struct ArgumentParser: CustomStringConvertible {
+    private var arguments: [ArgumentType]
+
+    private enum ArgumentType: CustomStringConvertible {
+        case argument(String)
+        case option(String)
```

Comment:
Shouldn't `flag` and `option` be the same case, just that there's a long and short form of the flag?